neo/bin/prompt.py

```diff
@@ -1150,8 +1152,9 @@ def main():
     cli = PromptInterface(fn_prompt_history)
 
     # Run things
-    #    reactor.suggestThreadPoolSize(15)
-    reactor.callInThread(cli.run)
+
+    d = reactor.callInThread(cli.run)
```

Comment:
do we want to add an `errBack` here incase some exception happens that we don't handle?